src/app/template.tsx

```diff
@@ -5,19 +5,22 @@ import Footer from "@/components/Footer";
 
 import "./global";
 
-export default function RootTemplate({
-  children,
-}: {
+interface RootTemplateProps {
   children: React.ReactNode;
-}) {
-  // const routeName = usePathname();
-  // const isLandingPagePath = routeName === "/";
+  offAnnouncement?: boolean;
+}
 
+/**
+ * @notice The `offAnnouncement` prop controls the visibility of the AnnouncementBar.
+ * @param {boolean} [offAnnouncement=true] - If true, the AnnouncementBar will be hidden.
+ */
+export default function RootTemplate({
+  children,
+  offAnnouncement = true,
```

Comment:
It would be clearer to use something like `announcementVisible = false` to  achieve the same effect. Positive naming usually makes the code easier to understand.

```js
// AnnouncementBar/index.tsx

if(announcementVisible){
   return <Box>...</Box>
}
return null
```

Reply:
Good point! Resolved